File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Cie10 SDK exists test

require_once __DIR__ . '/../cie10_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Cie10SDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Cie10 SDK exists test

require "minitest/autorun"
require_relative "../Cie10_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Cie10SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
